container_service_extension/rde/backend/cluster_service_2_x_tkgm.py

```diff
@@ -2033,8 +2036,8 @@ def _add_control_plane_nodes(
         for spec in vm_specs:
             spec['cust_script'] = templated_script.format(
                 vcd_host=vcd_host.replace("/", r"\/"),
-                org=org,
-                vdc=vdc,
+                org=org.get_name(),
```

Comment:
The changes in these two lines must be undone, right?

Reply:
As discussed, org.get_name() and vdc.name are the correct parameters